data/array/first item getter (with not found)/e408f545-5648-47e7-9392-6c39c0dd14d1.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "data/array/First item getter (with not found)",
```

Comment:
Better to extend the current "data/array/First item getter" with not found output port.